data_steward/cdr_cleaner/manual_cleaning_rules/remediate_basics.py

```diff
@@ -129,9 +130,13 @@
 FROM `{{project}}.{{incremental_dataset}}.{{table}}` inc_o
 JOIN `{{project}}.{{obs_id_lookup_dataset}}.{{new_obs_id_lookup}}` new_id
 ON inc_o.observation_id = new_id.source_observation_id
-WHERE inc_o.person_id IN (
-    SELECT person_id FROM `{{project}}.{{dataset}}.{{table}}` 
+{% if exclude_lookup_dataset and exclude_lookup_table %}
```

Comment:
@hiro-mishima am I assuming right that `exclude_lookup_dataset` and `exclude_lookup_table` corresponds to the names where AIAN participants list is available?

Reply:
Yes, that is correct, @ksdkalluri